test/tests/services/has-many-dissociator.test.js

```diff
@@ -0,0 +1,124 @@
+import mongoose from 'mongoose';
+import loadFixture from 'mongoose-fixture-loader';
+import Interface from 'forest-express';
+import mongooseConnect from '../../utils/mongoose-connect';
+import HasManyDissociator from '../../../src/services/has-many-dissociator';
+
+describe('service > has-many-dissociator', () => {
+  let TreeModel;
+  let LumberJackModel;
+
+  beforeAll(() => {
+    Interface.Schemas = {
+      schemas: {
+        LumberJack: {
+          name: 'LumberJack',
+          fields: [
+            {
+              field: '_id',
+              type: 'ObjectId',
+            },
+            {
+              field: 'name',
+              type: 'String',
+            },
+          ],
+        },
+        Tree: {
+          name: 'Tree',
+          fields: [
+            {
+              field: '_id',
+              type: 'ObjectId',
+            },
+            {
+              field: 'name',
+              type: 'String',
+            },
+            {
+              field: 'owners',
+              type: ['ObjectId'],
+              reference: 'LumberJack._id',
+            },
+          ],
+        },
+      },
+    };
+
+    return mongooseConnect()
+      .then(() => {
+        const LumberJackSchema = new mongoose.Schema({
+          _id: { type: 'ObjectId' },
+          name: { type: String },
+        });
+        const TreeSchema = new mongoose.Schema({
+          id: { type: 'ObjectId' },
+          name: { type: String },
+          owners: {
+            type: ['ObjectId'],
+            ref: 'LumberJack',
+          },
+        });
+
+        LumberJackModel = mongoose.model('LumberJack', LumberJackSchema);
+        TreeModel = mongoose.model('Tree', TreeSchema);
+
+        return Promise.all([LumberJackModel.remove({}), TreeModel.remove({})]);
```

Comment:
[MISC] You are doing this in the `beforeAll` and in the `beforeEach` 🤔 ? Why ?

Reply:
this was a copy-paste from `resource-remover.test.js` but it's true that it doesn't make much sense, I will keep it only in the `beforeEach`